stock_picking_import/wizards/stock_import_picking_wizard.py

```diff
@@ -0,0 +1,113 @@
+# © 2021 initOS GmbH
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+import base64
+import csv
+import io
+
+from odoo import _, api, fields, models
+from odoo.exceptions import UserError
+
+
+class StockPartialPickingImportWizard(models.TransientModel):
+    _name = "stock.partial.picking"
+    _description = _("Stock Partial Picking")
```

Comment:
Why 'partial' ?

Reply:
Hi @rousseldenis, I've added this because it is functionality.

Please find the attached image:

![stock_picking_import](https://user-images.githubusercontent.com/106082361/188886294-5f676c05-0817-43f2-8465-9ba965437568.png)

Reply:
Yes, I saw. Functionnally speaking, why 'partial' ? Can you explain a little bit ? My question due to the name 'stock_picking_import' which differs from.

Reply:
@rousseldenis  It is just a name given but as you know this module is creating a delivery order of SO using a importing CSV file.